packages/client/src/client/App.tsx

```diff
@@ -9,6 +9,7 @@ import { getMainApp } from "./main"
 import {
   registerCreditAcceptedNotifications,
   registerFxNotifications,
+  registerRFQRequestConfirmationNotifications,
```

Comment:
can we call it `registerCredit...` to keep with the asset class naming .. e.g. `registerCreditCreatedNotifications`
(or if you want to keep the RFQ in there - not a bad idea - add to both ..Created.. and ..Accepted..)

Reply:
We could just have `registerFXNotifications` and `registerCreditNotifications` methods, unless we need to call them individually somewhere?